tox.ini

```diff
@@ -5,53 +5,53 @@ skipsdist = true
 [testenv]
 deps = -r requirements-dev.txt
 commands_pre =
-    docker: docker exec es elasticsearch-plugin -s install analysis-icu
-    docker: docker restart es
+    docker: docker exec es-core elasticsearch-plugin -s install analysis-icu
+    docker: docker restart es-core
     python -m textblob.download_corpora
 commands =
-    pytest {posargs:--disable-warnings -v tests}
-passenv = SIMPLIFIED_*
+    pytest {posargs:--disable-warnings tests}
+passenv = SIMPLIFIED_* CI
```

Comment:
Pass the `CI` environment variable though from tox into the testing process. Pytest looks to see if this environment variable is set and doesn't truncate error output if it is. Github actions sets the CI env variable by default.